packages/javascript-sdk/src/http-client/index.ts

```diff
@@ -157,20 +162,15 @@ abstract class HttpClient {
     return headers;
   }
 
-  private static async stepIterator(
-    res: Response,
-    handleStep: HandleStep,
-    type: string,
-    tree: string,
-  ): Promise<void> {
+  private static async stepIterator(res: Response, handleStep: HandleStep): Promise<void> {
     const jsonRes = await res.json();
     const initialStep = new FRStep(jsonRes);
 
     // eslint-disable-next-line no-async-promise-executor
     return new Promise(async (resolve, reject) => {
       async function handleNext(step: FRStep): Promise<void> {
         const input = await handleStep(step);
-        const output = await FRAuth.next(input, { type, tree });
+        const output = await FRAuth.next(input, { tree: '', type: '' });
```

Comment:
Sure. We discussed this in the previous PR earlier this year (see conversation in [308](https://github.com/ForgeRock/forgerock-javascript-sdk/pull/308/files#r1068479053)). This is required because any `authIndexType` and `authIndexValue` params interfere with the authentication flow with advice after the first submission.

Reply:
ah ok thank you 👍